python/pyarrow/tests/test_plasma.py

```diff
@@ -30,6 +30,10 @@
 import pyarrow as pa
 
 
+# ignore all Plasma deprecation warnings in this file, we test that the
+# warnings are actually raised in test_plasma_deprecated.py
+pytestmark = pytest.mark.filterwarnings("ignore::DeprecationWarning")
```

Comment:
```suggestion
pytestmark = pytest.mark.filterwarnings("ignore:'Plasma:DeprecationWarning")
```

To make the filter more explicit for only the plasma related warnings, and not any other that we might need to see (although in the case of the plasma tests, we probably don't care too much since we are going to remove those anyway ..)